src/foam/nanos/controller/ApplicationController.js

```diff
@@ -406,6 +388,36 @@ foam.CLASS({
     // not navigate to a new screen. Typically for SubMenus
     function lastMenuLaunchedListener(m) {
       this.lastMenuLaunched = m;
+    },
+
+    // Get the most appropriate Theme object from the server and use it to
+    // customize the look and feel of the application.
+    async function fetchTheme() {
+      // TODO: Don't use a service. Just hit the DAO directly.
+      try {
+        this.theme = await this.client.themeService.getTheme(null, this.webApp);
+      } catch (err) {
+        this.notify(this.LOOK_AND_FEEL_NOT_FOUND, 'error');
+        console.error(err);
+        return;
+      }
+
+      this.useCustomElements();
+    },
+
+    // Use custom elements if supplied by the Theme.
```

Comment:
If you put comments inside the function then the FOAM doc tool can pick them up. Ex.:function useCustomElements() {
  /* Use custom elements if supplied by the Theme. */

Reply:
Does it pick up `// ...` style comments as well?

Reply:
Nvm, looks like it does.

Reply:
Resolved by 439b82623.